packages/common-common/src/redisCache.ts

```diff
@@ -1,7 +1,7 @@
 import { ConnectionTimeoutError, createClient, ReconnectStrategyError, SocketClosedUnexpectedlyError } from "redis";
-import { factory, formatFilename } from 'common-common/src/logging';
-import { REDIS_URL, VULTR_IP } from '../config';
-import { RedisNamespaces } from '../../shared/types';
+import { factory, formatFilename } from './logging';
+import { REDIS_URL, VULTR_IP } from 'commonwealth/server/config';
```

Comment:
Don't import from commonwealth into common-common.

Reply:
For replacing RedisNamespaces, we can use a string internally to RedisCache, unless we want to track within the cache all used namespaces.

For env vars, we should pass them into the init call.